packages/terra-base/src/Base.jsx

```diff
@@ -87,14 +87,14 @@ class Base extends React.Component {
 
     if (locale === undefined) {
       return (
-        <div {...customProps}>
+        <div className={['base', customProps.className]} {...customProps}>
```

Comment:
Is there styling associated with the `base` class? If not, when this is packed by webpack, the class name will not be hashed and will display as `base`, which means this class name is not locally scoped to this react module. If there are no styles associated, I would suggest a data attribute `terra-base-main`

Reply:
Let's update this to:
```suggestion
        <div data-terra-base {...customProps}>
```